packages/ui-components/src/Input/number-input.scss

```diff
@@ -0,0 +1,76 @@
+@import "../styles/tokens.scss";
+@import "./constants.scss";
+
+// remove default arrow spinner for input (number type)
+.input[type="number"]::-webkit-outer-spin-button,
+.input[type="number"]::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+  pointer-events: none;
+}
+
+// remove default arrow spinner for input -- Firefox browser
+.input[type=number] {
+  -moz-appearance: textfield;
```

Comment:
In Firefox, the spin buttons are being pushed out of the width of the input container by the `width: 100%` of the base input.

<img width="235" alt="Screen Shot 2020-05-21 at 12 29 33" src="https://user-images.githubusercontent.com/397448/82581627-d91cdd00-9b5e-11ea-9e4e-ce2f1e506162.png">

```suggestion
  -moz-appearance: textfield;
  padding-right: 0;
  width: calc(100% - 32px);
```

Reply:
Wow, missed testing for `NumberInput` on Firefox.
Found that `overflow-x` attribute fixes this case. Moreover, `NumberInput` might have or not the `prefix` icon so calculating width would be much complicated.

Also added `firefox-only` mixing to apply styles for Firefox only.

Reply:
Perfect, I didn’t consider the prefix for width calculation.